### ISSUE TITLE
Accessibility issue: Text on image contrast is too low

### ISSUE THREAD

**Issue by Nienke0905** [parley-messaging/ios-library#203]
Please see the issue for Android here: https://github.com/parley-messaging/android-library/issues/91

From our accessibility audit we got the following issue:

_The contrast ratio between the color of the normal text and the color of its background is too low. Text smaller than 18pt (24px) in regular font weight or 14pt (18.66px) in bold font is considered normal text. Ensure that all normal text has a contrast ratio of at least 4.5:1 against its background.

The date "15:20" does not have a fixed background color. Because the image is yellow it only has a contrast of 2:1._

![Image](https://github.com/user-attachments/assets/5a50e726-5be8-4ca7-bdd7-aa3da9f5ccbf)

It is currently possible to set the `imageInnerShadowStartColor` and `imageInnerShadowEndColor` in the Parley SDK, however that does not go very gradually, rather it almost immediately jumps to the end color. 

![Image](https://github.com/user-attachments/assets/0e457c87-f95e-45c6-ac0e-2b532bfb101e)

It would be good if the solution looks similar to Android's fix. Perhaps this could be fixed with a https://developer.apple.com/documentation/UIKit/UIVisualEffectView .

<img width="478" alt="Image" src="https://github.com/user-attachments/assets/9997b7b9-13c4-40e5-8ac9-4219c9744d1a" />

**Comment by alexkok** [comment]
Fixed in [4.2.9](https://github.com/parley-messaging/ios-library/releases/tag/4.2.9) 🚀 

_(Configure the `imageTimeBackgroundColor` in the `ParleyMessageViewAppearance` to configure the background color of the time)_

**Comment by Nienke0905** [comment]
Hi @alexkok ,

Thank you for the quick fix. 

Unfortunately, the checkmark also is required to have the background as it communicates something to the user. Would you please be able to extend the background to the checkmark? 

In addition, the designers would like to be able to give the corners a curve radius. Could you please add this?

**Comment by alexkok** [comment]
Hi @Nienke0905 , thanks for the response. We'll reopen to add this 👍

**Comment by alexkok** [comment]
Released in [4.2.10](https://github.com/parley-messaging/ios-library/releases/tag/4.2.10) 🚀
